requirements.txt

```diff
@@ -1,8 +1,10 @@
 batinfo==0.3
+-e git+https://github.com/bochecha/dbus-python.git@pipified-1.1.1#egg=dbus
```

Comment:
> You mean it's already on pypi even without setup.py? o_O

Even if it wasn't, it's still not very nice to take pypi/dbus-python when we're not dbus-python upstream.

But in any case, yes, it already is there:
https://pypi.python.org/pypi/dbus-python/

It's a very old version, though.

My guess is that upstream uploaded it there after being told that dbus-python should be on pypi, without knowing that it wouldn't work without a `setup.py` file (keep in mind they are OS-level, C hackers, not Python devs), and then when they realized it didn't work they just stopped sending newer versions.

I'm sure they'd be happy to upload newer version again after they add the `setup.py` upstream, if that ever happens.